src/Write.cpp

```diff
@@ -104,6 +105,12 @@ std::cout << folderPath_ << std::endl;
       return;
     }
   }
+  else if(fileEnding_ == "pcd"){
+    //Write pcd file
+    PointCloud<PointXYZRGBNormal> pclCloud;
+    fromROSMsg(*cloud, pclCloud);
+    pcl::io::savePCDFile (filePath.str(),pclCloud);
```

Comment:
You can omit the pcl::io namespace, as it is active due to the "using" above. Also, please use the same formatting as the rest of the code regarding spaces.

Reply:
Thanks for the feedback.
I just made the changes you suggested.
Can you check?
Thanks